patches/@firebase+messaging+0.12.13.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@firebase/messaging/dist/esm/index.esm2017.js b/node_modules/@firebase/messaging/dist/esm/index.esm2017.js
+index 612f735..070ea56 100644
+--- a/node_modules/@firebase/messaging/dist/esm/index.esm2017.js
++++ b/node_modules/@firebase/messaging/dist/esm/index.esm2017.js
+@@ -565,7 +565,7 @@ async function getPushSubscription(swRegistration, vapidKey) {
+         return subscription;
+     }
+     return swRegistration.pushManager.subscribe({
+-        userVisibleOnly: true,
++        userVisibleOnly: false,

```

Comment:
Chrome allows it since [121](https://developer.chrome.com/docs/extensions/how-to/integrate/web-push#silent_push), but the SDK doesn't support it yet.